apps/src/assetManagement/animationLibraryApi.js

```diff
@@ -91,26 +125,7 @@ export function createDefaultSpriteMetadata(listData) {
 // Regenerates the metadata for the default list of sprites in SpriteLab
```

Comment:
nit: this was already the behavior, but we should specify in this comment that we also upload the data to S3 in this function